src/main/java/bio/terra/janitor/service/cleanup/StairwayLoggingHooks.java

```diff
@@ -0,0 +1,74 @@
+package bio.terra.janitor.service.cleanup;
+
+import bio.terra.janitor.db.TrackedResourceId;
+import bio.terra.stairway.FlightContext;
+import bio.terra.stairway.HookAction;
+import bio.terra.stairway.StairwayHook;
+import java.time.Instant;
+import java.time.ZoneId;
+import java.time.format.DateTimeFormatter;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.slf4j.MDC;
+
+public class StairwayLoggingHooks implements StairwayHook {
+  private static final String FlightLogFormat =
+      "Operation: {}, flightClass: {}, flightId: {}, timestamp: {}";
+  private static final String StepLogFormat =
+      "Operation: {}, flightClass: {}, flightId: {}, stepIndex: {}," + "timestamp: {}";
+  private static final String TRACKED_RESOURCE_MDC_KEY = "trackedResourceId";
+  private static final Logger logger = LoggerFactory.getLogger(StairwayHook.class);
+
+  @Override
+  public HookAction startFlight(FlightContext context) {
+    logger.info(
+        FlightLogFormat,
```

Comment:
why is it important to re-use the same FLIGHT_LOG_FORMAT for startFlight and endFlight?

Reply:
I am not sure I understand this comment. This is to define the common static string variables then we can reuse.